tools/topology/sof-jsl-da7219.m4

```diff
@@ -18,6 +18,8 @@ include(`sof/tokens.m4')
 # Include Jasperlake DSP configuration
 include(`platform/intel/'PLATFORM`.m4')
 
+DEBUG_START
```

Comment:
Do we want this on master?

Reply:
@paulstelian97 This is the shared debug macro for all TPLG.